.github/workflows/mac.yml

```diff
@@ -123,7 +123,7 @@ jobs:
           NIGHTLY_REPO_TOKEN: ${{ secrets.NIGHTLY_REPO_TOKEN }}
           NIGHTLY_REPO: ${{ github.repository_owner }}/servo-nightly-builds
       - name: Build package for target
-        run: gtar -czf target.tar.gz target/${cargo_profile_name}/servo target/${cargo_profile_name}/*.dylib target/${cargo_profile_name}/lib/*.dylib resources
+        run: gtar -czf target.tar.gz target/${cargo_profile_name}/servo target/${cargo_profile_name}/lib/*.dylib resources
```

Comment:
It seems that this change is unrelated now that the `cargo_profile_name` change landed?

Reply:
This PR needs rebase.